### PR TITLE
[FluentInput] Set last modified date

### PR DESCRIPTION
**Pull request by maNobly**
# Pull Request

## 📖 Description
Making the property LastModified from the IBrowserFile available on the the FluentInputEventArgs object, so it can be accessed.

### General

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fluentui-blazor/blob/master/docs/contributing.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Comment by dvoituron** [comment]
Have you verified that this `File.LastModified' property is also available in a WASM project?

File: src/Core/Components/InputFile/FluentInputFileEventArgs.cs [hunk 1]
```diff
@@ -42,6 +42,11 @@ public class FluentInputFileEventArgs : EventArgs
     /// </summary>
     public string ContentType { get; internal set; } = string.Empty;
 
+    /// <summary>
+    /// Gets the last modified date of the current file in an upload process.
+    /// </summary>
+    public DateTimeOffset LastModified { get; internal set; } = default!;
```
Comment (dvoituron) [inline src/Core/Components/InputFile/FluentInputFileEventArgs.cs:48]: Can you add a Unit Test to validate this new feature?
You will find some examples in the `FluentInputFileTests.razor` file

Comment (maNobly) [reply src/Core/Components/InputFile/FluentInputFileEventArgs.cs:48]: Yes, I have added unit tests now

**Comment by maNobly** [comment]
> Have you verified that this `File.LastModified' property is also available in a WASM project?

Yes, I have also tested in WASM project and the prop is available.

**Review by dvoituron — APPROVED** [review]

**Pull request merged**